sdks/workflow-browser-sdk/src/lib/workflow-browser-sdk.ts

```diff
@@ -229,7 +229,7 @@ export class WorkflowBrowserSDK {
   }
 
   sendEvent(event: WorkflowEventWithoutState) {
```

Comment:
Let's add the `async` keyword here as well now that `sendEvent` is async in `core`.

Reply:
fixed, also made the same change in `workflow-node-sdk.ts`